app/src/tests/Appointment.test.tsx

```diff
@@ -1,12 +1,14 @@
 import classData from 'data/clean.json';
-import { parseCalendarJSON } from 'utils/utils';
-import { getClassById } from 'components/organisms/Appointment';
+import { parseCalendarTeacher, parseCalendarCourse } from 'utils/utils';
+
+import getTeacherById from 'components/organisms/Appointment';
 
 test('Testing getClassByID', () => {
-  const expected = parseCalendarJSON(JSON.parse(JSON.stringify(classData.slice(0, 1))))[0];
-  const res = getClassById(expected.courseId);
+  const expected = parseCalendarTeacher(JSON.parse(JSON.stringify(classData.slice(0, 1))))[0];
+  const courseData = parseCalendarCourse(JSON.parse(JSON.stringify(classData.slice(0, 1))))[0];
+  const res = getTeacherById(courseData.teacherId);
 
-  expect(res).toEqual(expected);
+  expect(res).toEqual(expected.id);

```

Comment:
either change to res and expected or res.id and expected.id. I like the former better as its testing the actual output.